frontend/src/locales/de.json

```
@@ -224,6 +230,7 @@
   "navigation": {
     "admin_area": "Adminbereich",
     "community": "Gemeinschaft",
+    "info": "Info",
```

```suggestion
    "info": "Information",
```

Is much better in my eyes!